include/enums.hpp

```diff
@@ -14,7 +14,7 @@ typedef enum latDim_t {
 extern const char latDim_names[];
 
 typedef enum LoopExtractLoopType_t {
-  LOOP_EXTRACT_LOOP_TYPE_NAIVE = 1,
+  LOOP_EXTRACT_LOOP_TYPE_NAIVE = 0,
```

Comment:
the last element should thus be called NTYPES rather than NTYPES_PLUS_ONE, right?